presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/ForwardingJdbcClient.java

```diff
@@ -45,14 +45,14 @@ public static JdbcClient of(Supplier<JdbcClient> jdbcClientSupplier)
         return new ForwardingJdbcClient()
         {
             @Override
-            protected JdbcClient delegate()
+            public JdbcClient delegate()
             {
                 return requireNonNull(jdbcClientSupplier.get(), "jdbcClientSupplier.get() is null");
             }
         };
     }
 
-    protected abstract JdbcClient delegate();
+    public abstract JdbcClient delegate();
```

Comment:
why?

i see this is used in a test code, but this should not warrant a change in the non-test code.
please revert

Reply:
To avoid that I added `quoted` method to `JdbcClient`. I think the fit there as they capture SQL dialect.